gfc/gfc/settings.py

```diff
@@ -31,7 +31,8 @@
     "django-insecure-)2rxjsa3d&2d83qxnyjyca(d(kl=tt6g*h&*et!-u$fa-w94_j'")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG: bool = False
+# TODO: Make false
```

Comment:
Does this need to be removed?

Reply:
It seems to remove access to the static folder on the site when set to false